### PR TITLE
Include tests in release source distribution

### PR DESCRIPTION
**Pull request by sbraz**
Could you include tests in PyPI releases? This facilitates packaging for certain distributions (at least Debian and Gentoo rely on it).

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,3 +1,4 @@
 include src/geventhttpclient/*.py
 include ext/*.c
 include ext/*.h
+recursive-include src/geventhttpclient/tests
```
Comment (rmohr) [inline MANIFEST.in]: Isn't there a star at the end of the line missing?
```
recursive-include src/geventhttpclient/tests *
```

**Comment by sbraz** [comment]
You're absolutely right, sorry! I re-pushed the correct MANIFEST.in.

**Comment by rmohr** [comment]
LGTM

**Comment by rmohr** [comment]
@sbraz Thanks. Are you going to package this library for gentoo?

**Comment by sbraz** [comment]
Yes, indeed :) I noticed that dulwich relies on it for tests so I thought it would be nice to package it.

**Pull request merged**